.github/workflows/ci.yml

```diff
@@ -1,8 +1,26 @@
-name: Cypress Tests
+name: Coderadio-client ci
 
 on: [push, pull_request]
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-16.04
```

Comment:
Apologies for not mentioning this with the last batch, but I do not think we should be using a version that is not longer receiving maintenance updates.

Unless there is some sort of penalty for doing so, I suggest going with `20.04`.

Reply:
I used that version because of the existing comment on cypress-run job that was saying that ubuntu-16.04 was the version of the build environment. I am ok also for using ubuntu 20.04 (or ubuntu-latest)
@raisedadead what do you think ?